styles/index.css

```diff
@@ -76,7 +75,7 @@
 .honeymoon-description-text{
     font-family: 'Poppins', sans-serif;
     color: #000000;
-    width: 580px;
+    width:600px;
```

Comment:
html `<br>` kesme etiketin, kullandığın için width vermene gerek kalmiycaktır

Reply:
Düzenlendi.